external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaMicroBatchReader.scala

```diff
@@ -299,13 +299,13 @@ private[kafka010] class KafkaMicroBatchReader(
   }
 }
 
-/** A [[DataReaderFactory]] for reading Kafka data in a micro-batch streaming query. */
+/** A [[ReadTask]] for reading Kafka data in a micro-batch streaming query. */
 private[kafka010] case class KafkaMicroBatchDataReaderFactory(
```

Comment:
@rdblue . The re-naming of this PR seems to be partial. Could you replace `KafkaMicroBatchDataReaderFactory` with `KafkaMicroBatchReadTask` together in this PR? I guess there will be more occurrences like this.

Reply:
This fixes the API, not implementations, and it already touches 30+ files.

I'd rather not fix the downstream classes for two reasons. First, to avoid this becoming really large. Second, we need to be able to evolve these APIs without requiring changes to all implementations. This is still evolving and if we need to update 20+ implementations to make simple changes, then I think that's a problem.

Reply:
Yes. This kind of changes always become unnecessarily big. Since this PR turns the master branch into an inconsistent state, could you make a JIRA issue for the remaining tasks which this PR avoids? Then, someone else can help Apache Spark become more consistent later eventually in Apache Spark 2.4 (or 3.0) timeframe.
> I think we should avoid requiring changes to make everything consistent

Reply:
Sure, good idea.